### PR TITLE
Task 2

### PR DESCRIPTION
**Pull request by captain2b**

### CODE REVIEW

File: src/app/courses/components/courses-page/courses-page.component.less [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  .header{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+  .loadMore{
+    display: flex;
+    width: 200px;
+    margin: auto;
+  }
```
Comment (dKab) [inline src/app/courses/components/courses-page/courses-page.component.less]: This isn't an error per se, but I would recommend to avoid putting everything inside :host { }  block. It adds one level of nesting which is generally not good. And it doesn't serve any practical purpose, because styles are scoped to the component host anyway... So, as a rule of thumb I'd place all css rules that are not related to host element separately.

File: src/app/courses/components/courses-item/courses-item.component.ts [hunk 2]
```diff
@@ -1,15 +1,21 @@
...
-  constructor() { }
-
-  ngOnInit() {
+  onDelete() {
+    this.delete.emit(this.courseItem.id);
+  }
+  getformattedDuration(): string {
+    const h = Math.floor(this.courseItem.duration / 60);
+    const m = this.courseItem.duration % 60;
+    return h + 'h' + m + 'min';
```
Comment (dKab) [inline src/app/courses/components/courses-item/courses-item.component.ts]: Please use full words for variable names, even for obvious ones like these. One letter variable name is a serious code smell. And template string es6 syntax should be used instead of string concatenation - it's more elegant and expressive.

File: src/app/courses/components/courses-page/courses-page.component.less (hunk 1 above)
Comment (captain2b) [reply src/app/courses/components/courses-page/courses-page.component.less]: fixed thanks

File: src/app/courses/components/courses-item/courses-item.component.ts (hunk 2 above)
Comment (captain2b) [reply src/app/courses/components/courses-item/courses-item.component.ts]: fixed
